Makefile

```diff
@@ -476,6 +511,10 @@ manager-kubeadm-bootstrap: ## Build the kubeadm bootstrap manager binary into th
 manager-kubeadm-control-plane: ## Build the kubeadm control plane manager binary into the ./bin folder
 	go build -trimpath -ldflags "$(LDFLAGS)" -o $(BIN_DIR)/kubeadm-control-plane-manager sigs.k8s.io/cluster-api/controlplane/kubeadm
 
+.PHONY: manager-capd
+manager-capd: ## Build the kubeadm control plane manager binary into the ./bin folder
+	cd $(CAPD_DIR); go build -trimpath -ldflags "$(LDFLAGS)" -o ../../../$(BIN_DIR)/capd-manager sigs.k8s.io/cluster-api/test/infrastructure/docker
```

Comment:
You asre right, that bin dir was **not** in top-level before.

It was at `test/infrastructure/docker/bin`.

All I've found is: it is used in `scripts/ci-build.sh` which is only used in the test-infra jobs:

* pull-cluster-api-build-main
* pull-cluster-api-build-release-0-3
* pull-cluster-api-build-release-0-4
* pull-cluster-api-build-release-1-0
* pull-cluster-api-build-release-1-1

When running the other `make manager-*` targets, the output is always in `${top-level-repo-dir}/bin`. So this would align with building the other binaries.

Reply:
Yup top-level / consistency sounds better